### DISCUSSION TITLE
沉浸式翻译使用的翻译模型是什么

### DISCUSSION THREAD

**Discussion by wwfcnu** [immersive-translate/immersive-translate#632] [Help]
### 概述 | Summary

想问一下，沉浸式翻译使用的翻译模型是什么，是开源的吗

**Comment by reece00** [comment]
还模型，玩ai入脑了？要不你先下个扩展看一眼？

**Comment by wwfcnu** [comment]
也就是说，它只是集成了一些翻译接口是吧



| |
***@***.***
|
|
***@***.***
|




---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2023年06月28日 16:46 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [immersive-translate/immersive-translate] 沉浸式翻译使用的翻译模型是什么 (Discussion #632) |

还模型，玩ai入脑了？要不你先下个扩展看一眼？

**Comment by Ayx03** [comment]
使用 OpenAI 自定义 API Key 时可以选择模型或输入自定义模型名称，可以自定义 API 接口地址，理论上应该可以使用 OpenAI-compatible 的其他 API，包括本地 API
![image](https://github.com/immersive-translate/immersive-translate/assets/75155322/282f1ff8-dc30-456c-bacb-dacd730698a5)
